src/modifiers/identity.py

```diff
@@ -0,0 +1,14 @@
+from talon import Module
+
+mod = Module()
+
+
+mod.list(
+    "cursorless_identity",
+    desc="Cursorless modifier for identity. Use to break inference.",
```

Comment:
Sounds a bit funny this way, as if the user is trying to break cursorless 😄
```suggestion
    desc="Cursorless modifier for identity. Use to break inference chain.",
```